Template/README.md

```diff
@@ -1 +1,18 @@
-# {PROJECT}
\ No newline at end of file
+# {PROJECT}
+
+### Installation
+
+**CocoaPods:**
+Add the line `pod "{PRODUCT}"` to your `Podfile`
+
+**Carthage:**
+
+Add the line `github "{URL}/{PRODUCT}"` to your `Cartfile`
```

Comment:
This won't be correct, for example given the `URL` `https://github.com/johnsundell/unbox`, this will resolve into:
```
github "https://github.com/johnsundell/unbox/unbox"
```
It should be:
```
github "johnsundell/unbox"
```

I think you'll need to introduce a new template property for this. Perhaps `CARTHAGE_PATH`?